kubernetes/yamls/development/web/ingress-tls.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: web
+  namespace: web
+  annotations:
+    kubernetes.io/tls-acme: "true"
+    kubernetes.io/ingress.class: "nginx"
+spec:
+  tls:
+  - hosts:
+    - staging.loklak.org
+    secretName: loklak-api-tls
+  rules:
+  - host: staging.loklak.org
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: api-server
```

Comment:
You are pointing this to a service named as `api-server`. But there is no service named as `api-server` in the `web` namespace.